packages/gatsby-www-components/.gitignore

```diff
@@ -0,0 +1 @@
+/*.js
```

Comment:
Why are you adding all js files to the gitignore? Was this something you added in debugging that got accidentally merged?

Reply:
I don't remember why it is there but it seems unnecessary, so I removed it.